### PR TITLE
redo camera and add new tif loader

### PR DESCRIPTION
**Pull request by aafkegros**
I had issues trying to add a screenshot of camera values with/without keyframes, because people will assume that they need to take over these values directly

### CODE REVIEW

File: lessons/1e_camera.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+To use the `Render` mode of rendering, we have to add a camera. This can be done easiest in the `Layout` tab. 
+Here press `Add` or `Shift+A` and add a camera.
+
+You can look through the current `Active Camera` by clicking the camera icon in the `3D viewport` panel (`Active Camera` is settable in the `Outliner` panel by clicking the green camera, if only one camera is present, this is the active camera)
+
+ <img src="../figures/view through camera.png" width="200"/>
+
+**For an easy top view:**
+In `Object Properties` set the camera `Location` to 0 in `X`/`Y` and above the volume in `Z`, and `Rotation` to 0 for all.
```
Comment (k-dominik) [inline lessons/1e_camera.md:12]: I find it easier to click on the `Z` axis, adjust zoom and do the align camera to viewport...

File: lessons/1e_camera.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+Then in `Object Data Properties` set the camera to `Orthographic` and the `Orthographic Scale` to encompass the volume. 
+
+ <img src="../figures/orthographic camera.png" width="200"/>
+
+This is similar to a Z projection, alghough the precise type of Z projection depends on the implemented shaders/materials.
+<details><summary>Orthographic vs Perspective cameras</summary> Orthographic cameras show all objects at the same scale, in perspective cameras, objects which are far away are smaller than those nearby. Orthographic projections are standard in microscopy (for Z projections, for example). Perspective projections are default in many other cases, as this is how we view the world usually. </details>
+
+\
+**Making other angles of view:**
+Here the location and rotation depends a lot more on your data :D. You can move the camera by the input values in `Object Properties` of the camera, or by pressing `G` and moving the mouse (constrain movement along an axis with `X` `Y` `Z` keys). `Rotation` and `Scale` can be done similarly with `R` and `S`.
```
Comment (k-dominik) [inline lessons/1e_camera.md]: I had very little success moving the camera around. Also still to me it's unclear what scale would do?!

File: lessons/1e_camera.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+Snap `Active Camera` to Viewport location with `Alt` `Ctrl` `Numpad 0` (check `Emulate Numpad` in `Preferences > Input` if you do not have a numpad), or alternatively, via the menu _View_ -> _Align View_ -> _Align Active Camera to View_.
+
+## Output
+
+Change the pixel number in x and y of the cameras under `Output Properties`, and with this the aspect ratio. Here you can also find where animations are being stored.
+
+Render an image with `Render > Render Image` or `F12`.
+
+## Animation
```
Comment (k-dominik) [inline lessons/1e_camera.md]: Consider moving Animation to a dedicated lesson, just to have focused context. First people do _one_ pretty picture, later a movie.

File: lessons/1f_new_tif_loader.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
+# New `tif loader`
+
+The version of `tif loader` we installed in this tutorial is actually not the current version! For didactic purposes, we wanted to show you how to make your own Materials and get used to the Blender user interface. We will now install the latest version of the `tif loader`.
+
+For this we have to `Remove` our current version, which is done in the same `Edit > Preferences > Add-Ons` tab as we installed in:
+- unfold the `tif loader` details
```
Comment (k-dominik) [inline lessons/1f_new_tif_loader.md]: adding a screenshot here probably would help

**Review by k-dominik — COMMENTED** [review]
great that you have added all the new part so quickly!

General comment: for future reviewability of changes I usually opt to having each sentence in a new line in markdown. This will still render as a single paragraph, but changes to sentences later on will have more focus in the diff. Of course one could also use `--word-diff` but who does :)

Otherwise I only had minor comments and think it would also be fine to merge as is :)

File: lessons/1e_camera.md (hunk 2 above)
Comment (aafkegros) [reply lessons/1e_camera.md]: fixed by having snap to view as initial

File: lessons/1e_camera.md (hunk 3 above)
Comment (aafkegros) [reply lessons/1e_camera.md]: done

**Pull request merged**